pavement.py

```diff
@@ -64,25 +65,40 @@ def live():
     server.serve(root='_build/html', open_url_delay=True)
 
 
-ROOT = path('.').abspath()
-ASSETS = path('assets').abspath()
-
-
 @task
 def assets_init():
-    path('assets').makedirs()
+    # create _build path assets for generated contents
+    path('_build').makedirs()
```

Comment:
you should be able to make
path('_build/html').makedirs_p()
path('_build/intermediate').makedirs_p()